corgidrp/darks.py

```diff
@@ -832,11 +871,18 @@ def build_synthesized_dark(dataset, noisemaps, detector_regions=None, full_frame
             Cdq = noise_maps.dq[1]
 
         # get from one of the noise maps and modify as needed
-        prihdr = noise_maps.pri_hdr
-        exthdr = noise_maps.ext_hdr
+        prihdr = noise_maps.pri_hdr.copy()      # Ensure header is copied
+        exthdr = noise_maps.ext_hdr.copy()
         errhdr = noise_maps.err_hdr
-        exthdr['NAXIS1'] = Fd.shape[0]
-        exthdr['NAXIS2'] = Fd.shape[1]
+
+        # Check and modify NAXIS1 and NAXIS2 in the appropriate header
```

Comment:
Yes, in all the files I have access to (the ones currently on Box and the ones I have separately from TVAC), NAXIS is in the primary header. Without this modification, build_synthesized_dark does not work on real data files because the FITS cards get out of place when trying to save a new value for NAXIS1 or NAXIS2 in the extension header.

Reply:
Doesn't the setup code put the TVAC calibration files into the corgidrp format and save it? If so, shouldn't the NoiseMaps used here be the ones saved in the corgidrp format (which should always have the data and thus the NAXIS keywords in the ext_hdr)?